code/modules/antagonists/cult/team_cult.dm

```diff
@@ -0,0 +1,485 @@
+
+	if(!display_members)
+		return
+	var/list/cult = get_cultists(separate = TRUE)
+	var/total_cult = cult[1] + cult[2]
+
+	var/overview = "<span class='cultitalic'><br><b>Current cult members: [total_cult]"
+	if(!cult_ascendant)
+		var/rise = rise_number - total_cult
+		var/ascend = ascend_number - total_cult
+		if(rise > 0)
+			overview += " | Conversions until Rise: [rise]"
+		else if(ascend > 0)
+			overview += " | Conversions until Ascension: [ascend]"
+	to_chat(M, "[overview]</b></span>")
+
+	if(cult[2]) // If there are any constructs, separate them out
+		to_chat(M, "<span class='cultitalic'><b>Cultists:</b> [cult[1]]")
+		to_chat(M, "<span class='cultitalic'><b>Constructs:</b> [cult[2]]")
+
+/datum/team/cult/proc/create_next_sacrifice()
+	var/datum/objective/sacrifice/obj_sac = objective_holder.add_objective(/datum/objective/sacrifice)
+	if(!obj_sac.target)
+		objective_holder.remove_objective(obj_sac)
+		ready_to_summon()
+		return
+	return obj_sac
+
+/datum/team/cult/proc/current_sac_objective() //Return the current sacrifice objective datum, if any
+	var/list/presummon_objs = objective_holder.get_objectives()
+	if(cult_status == NARSIE_DEMANDS_SACRIFICE && length(presummon_objs))
+		var/datum/objective/sacrifice/current_obj = presummon_objs[length(presummon_objs)]
+		if(current_obj.sacced)
+			return create_next_sacrifice()
+		if(istype(current_obj))
+			return current_obj
+
+/datum/team/cult/proc/is_sac_target(datum/mind/mind)
+	var/datum/objective/sacrifice/current_obj = current_sac_objective()
+	return istype(current_obj) && current_obj.target == mind
+
+/datum/team/cult/proc/find_new_sacrifice_target(datum/mind/mind)
+	var/datum/objective/sacrifice/current_obj = current_sac_objective()
+	if(!current_obj)
+		return FALSE
+	if(!current_obj.find_target())
+		ready_to_summon()
+		return FALSE
+	for(var/datum/mind/cult_mind in members)
+		if(cult_mind && cult_mind.current)
+			to_chat(cult_mind.current, "<span class='danger'>[GET_CULT_DATA(entity_name, "Your god")]</span> murmurs, <span class='cultlarge'>Our goal is beyond your reach. Sacrifice [current_obj.target] instead...</span>")
+	return TRUE
+
+/datum/team/cult/proc/successful_sacrifice()
+	var/datum/objective/sacrifice/current_obj = current_sac_objective()
+	if(!istype(current_obj))
+		return
+	current_obj.sacced = TRUE
+	sacrifices_done++
+	if(sacrifices_done >= sacrifices_required)
+		ready_to_summon()
+		return
+
+	var/datum/objective/sacrifice/obj_sac = create_next_sacrifice()
+	if(!obj_sac)
+		return
+
+	for(var/datum/mind/cult_mind in members)
+		if(cult_mind && cult_mind.current)
+			to_chat(cult_mind.current, "<span class='cult'>You and your acolytes have made progress, but there is more to do still before [GET_CULT_DATA(entity_title1, "The Dark One")] can be summoned!</span>")
+			to_chat(cult_mind.current, "<span class='cult'>Current goal: [obj_sac.explanation_text]</span>")
```

Comment:
Seems to me like you make use of this sort of code a fair bit, perhaps you could make a helper that takes in a list of strings and distributes them out to all of the members?